### PR TITLE
docs: first draft of readme

### PR DESCRIPTION
**Pull request by Kuan-Zher**

### CODE REVIEW

**Review by DevOps-2023-TeamA — DISMISSED** [review]
Approved.

**Review by DanielCheahSJ — APPROVED** [review]
Approved.

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+```
+python -m http.server [Port Number]
+```
+
+4. Open your browser and ensure that the website is open
+
+## Troubleshooting
+
+## Maintainers
```
Comment (jiachenyee) [inline README.md:48]: Should we put our student IDs?

Comment (jiachenyee) [reply README.md:48]: cc: @isabellepakyishan

**Review by jiachenyee — CHANGES_REQUESTED** [review]

**Review by jiachenyee — APPROVED** [review]
Thanks for the change, lgtm!

**Pull request merged**